noir-r1cs/src/cli/cmd/prove.rs

```diff
@@ -12,7 +12,11 @@ use {
 #[derive(FromArgs, PartialEq, Eq, Debug)]
 #[argh(subcommand, name = "prove")]
 pub struct Args {
-    /// path to the compiled Noir program
+    /// path to the compile Noir program
+    #[argh(positional)]
+    program_path: PathBuf,
```

Comment:
The inputs to the prover are a `.nps` file and a `.toml` file. Nothing more, nothing less. Please add things to NoirProofScheme as necessary.